minttea/event.ml

```diff
@@ -1,5 +1,8 @@
 open Riot
 
+type modifier = Ctrl
+type key_event = { key : string; modifier : modifier option }
```

Comment:
Could we move the `modifier` up to the `KeyDown` constructor? 

```ocaml
type t =
 | KeyDown of key * modifier
```

I think this would allow you to pass in modifier for the special keys too, while keeping all the pattern matching ergonomic:

```ocaml
let update t e = 
  match e with
  | KeyDown (Key "h" | Left, Ctrl) -> prev_page t
  | KeyDown (Key "l" | Right, None) -> next_page t
```

Reply:
Didn't think of this, I think this is way better and more ergonomic, will add these changes